internal/entitlement/httpdriver/parser.go

```diff
@@ -0,0 +1,115 @@
+package httpdriver
+
+import (
+	"fmt"
+
+	"github.com/openmeterio/openmeter/api"
+	"github.com/openmeterio/openmeter/internal/entitlement"
+	booleanentitlement "github.com/openmeterio/openmeter/internal/entitlement/boolean"
+	meteredentitlement "github.com/openmeterio/openmeter/internal/entitlement/metered"
+	staticentitlement "github.com/openmeterio/openmeter/internal/entitlement/static"
+	"github.com/openmeterio/openmeter/pkg/convert"
+)
+
+type parser struct{}
+
+var Parser = parser{}
+
+func (parser) ToMetered(e *entitlement.Entitlement) (*api.EntitlementMetered, error) {
+	metered, err := meteredentitlement.ParseFromGenericEntitlement(e)
+	if err != nil {
+		return nil, err
+	}
+
+	return &api.EntitlementMetered{
+		CreatedAt:       &metered.CreatedAt,
+		DeletedAt:       metered.DeletedAt,
+		FeatureId:       metered.FeatureID,
+		Id:              &metered.ID,
+		IsUnlimited:     convert.ToPointer(false), // implement
+		IssueAfterReset: metered.IssuesAfterReset,
```

Comment:
As far as I see it's not implemented just stored, right?

Reply:
Also if this is used to auto issue metered entitlements after auto resets we should have the same solution for string and bools, or what's the plan here? cc @hekike

Reply:
> As far as I see it's not implemented just stored, right?

yes, it will be in a followup
> Also if this is used to auto issue metered entitlements after auto resets we should have the same solution for 
string and bools, or what's the plan here?

wdym same solution for static and bools? what would that mean?